tests/ecmp/test_fgnhg.py

```diff
@@ -22,6 +26,7 @@
 PREFIX_IPV6_LIST = [u'fc:05::/128', u'fc:06::/128', u'fc:07::/128']
 FG_ECMP_CFG = '/tmp/fg_ecmp.json'
 USE_INNER_HASHING = False
+WARM_BOOT_TESTING = True
```

Comment:
@kktheballer can we make it a command line option?

Reply:
Actually I will just make this enabled by default if that sounds ok

Reply:
I have taken this out because I think we want warm boot testing by default